docs/usage/golang.md

```diff
@@ -38,5 +38,5 @@ Renovate will commit all files changed within the `vendor/` folder.
 
 #### Go binary version
 
-Currently, Renovate will try to keep up with the very latest version of `go`, and it is not configurable.
-It is planned though to support a configurable version of `go` per-repository soon.
+By default, Renovate will keep up with the very latest version of `go`.
+You can configure the `go` version that Renovate uses by (insert action reader can take to configure this).
```

Comment:
We maybe need to explain how to configure the version?

Reply:
https://blog.golang.org/using-go-modules#TOC_2.

You need to add `go 1.13` to `go.mod` file. (version can be changed to whatever is required) 🙃